### ISSUE TITLE
Audit form fields that should be fieldsets

### ISSUE THREAD

**Issue by emilyjablonski** [bloom-housing/bloom#4706]
Sets of related fields should have a legend and a fieldset, following example 3 in this accessibility documentation.

This would include fields like:
Create account - first name, middle name, last name
Application - address fields
